### PR TITLE
config: let the cert-allowed-cn same with other components

### PR DESCRIPTION
**Pull request by nolouch**
Signed-off-by: nolouch <nolouch@gmail.com>

### What problem does this PR solve? 
TiDB, TiKV Supports multiple CNs, but we depend on etcd, let the config be consisted firstly.

### What is changed and how it works?
changed config items.

### Check List 

Tests 

 - Manual test

### CODE REVIEW

File: pkg/grpcutil/grpcutil.go [hunk 1]
```diff
@@ -55,6 +60,20 @@ func (s SecurityConfig) ToTLSConfig() (*tls.Config, error) {
 	return tlsConfig, nil
 }
 
+// GetOneAllowedCN only gets the first one CN.
+func (s SecurityConfig) GetOneAllowedCN() (string, error) {
+	allowedCN := ""
+	i := len(s.CertAllowedCN)
+	switch {
+	case i > 1:
+		return "", errors.New("Currently only supports one CN")
```
Comment (rleungx) [inline pkg/grpcutil/grpcutil.go]: Do we need to return one and give the error?

Comment (nolouch) [reply pkg/grpcutil/grpcutil.go]: I think no need.

File: pkg/grpcutil/grpcutil.go [hunk 2]
```diff
@@ -55,6 +60,20 @@ func (s SecurityConfig) ToTLSConfig() (*tls.Config, error) {
 	return tlsConfig, nil
 }
 
+// GetOneAllowedCN only gets the first one CN.
+func (s SecurityConfig) GetOneAllowedCN() (string, error) {
+	allowedCN := ""
```
Comment (HunDunDM) [inline pkg/grpcutil/grpcutil.go]:
```go
	switch len(s.CertAllowedCN) {
	case 1:
		return s.CertAllowedCN[0], nil
	case 0:
		return "", nil
	default:
		return "", errors.New("Currently only supports one CN")
	}
```
Would it be more readable?

**Comment by nolouch** [comment]
will break the compatibility between https://github.com/pingcap/pd/pull/2241 and  https://github.com/pingcap/pd/pull/2251.

**Pull request merged**